src/InputContainer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import Input from './Input';
+
+import { updateTaskTitle, addTask } from './actions';
+
+export default function InputContainer() {
+  const { taskTitle } = useSelector((state) => ({
+    taskTitle: state.taskTitle,
+    tasks: state.tasks,
+  }));
```

Comment:
`taskTitle`밖에 사용하기 있지 않으니 `tasks`는 지워줘도 될 것 같아요

```suggestion
  const { taskTitle } = useSelector((state) => ({
    taskTitle: state.taskTitle,
  }));
```

Reply:
넵넵!! 피드백에 대한 수정은 과제2 제출하고 나서 해도 될까요??ㅎㅅㅎ

Reply:
네넵 ㅎㅎ